src/lib/mina.ts

```diff
@@ -340,6 +346,7 @@ interface Mina {
   ) => ReturnType<typeof Fetch.fetchEvents>;
   getActions: (
     publicKey: PublicKey,
+    actionHashes: ActionHashes,
```

Comment:
I would make this argument optional, otherwise users have to pass in an empty object